examples/spring-boot-webflux/src/main/java/example/springframework/boot/webflux/HelloConfiguration.java

```diff
@@ -0,0 +1,44 @@
+package example.springframework.boot.webflux;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.linecorp.armeria.client.Client;
+import com.linecorp.armeria.client.ClientFactoryBuilder;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.logging.AccessLogWriter;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.spring.ArmeriaServerConfigurator;
+import com.linecorp.armeria.spring.web.reactive.ArmeriaClientConfigurator;
+
+import io.netty.handler.ssl.util.InsecureTrustManagerFactory;
+
+/**
+ * An example of a configuration which provides beans for customizing the server and client.
+ */
+@Configuration
+public class HelloConfiguration {
+
+    /**
+     * A user can configure a {@link Server} by providing an {@link ArmeriaServerConfigurator} bean.
+     */
+    @Bean
+    public ArmeriaServerConfigurator armeriaServerConfigurator() {
+        // An example which adds a LoggingService decorator to the ServerBuilder in order to log every request.
+        return builder -> builder.decorator(LoggingService.newDecorator())
```

Comment:
How about adding an additional example that shows:

- a user can bind a gRPC or Thrift service as a comment
- HAPROXY support
- DocService

..?

e.g.

```
builder -> {
    builder.serviceUnder("/docs", new DocService())
           .decorator(...)
           .accessLogWriter(...);

    // You can also bind an asynchronous RPC services such as gRPC and Thrift:
    // builder.service(new GrpcServiceBuilder() .... build());
    // builder.service(THttpService.of(...));

    return builder;
}
```